ecosystem/sep-0005.md

```diff
@@ -69,6 +81,12 @@ Mnemonic (12 words):
 illness spike retreat truth genius clock brain pass fit cave bargain toe
 ```
 
+Entropy:
+
+```
+713a3ae074e60e56c6bd0557c4984af1
```

Comment:
I suggest placing `Entropy` at the top because the BIP 39 phrase is generated from entropy, although they can be converted into each other.